app/src/pages/domain/index.js

```diff
@@ -42,7 +42,11 @@ export default class Domain extends Component {
 
   onSubmit = (e) => {
     e.preventDefault()
-    const value = this.state.tab === 'grape' ? grapeDomain : parse(this.state.value).host || this.state.value
+    let value = grapeDomain
+    if (this.state.tab !== 'grape') {
+      const parsed = parse(this.state.value)
+      value = parsed.slashes ? parsed.host : this.state.value
```

Comment:
Maybe we could add an extra layer of validation here in case `slashes` is `false`

```js
let parsed = parse(this.state.value)
if (!parsed.slashes)
  parsed = parse(`http://${this.state.value.replace(/^\:?\/{0,2}/g, '')}`)
value = parsed.host
```

but probably it is overkill